src/sagas/actionSaga.ts

```diff
@@ -8,7 +8,8 @@ import { postSystemIntakeAction } from 'types/routines';
 
 function postSystemIntakeActionRequest(formData: Action) {
   return axios.post(
-    `${process.env.REACT_APP_API_ADDRESS}/system_intake/${formData.intakeId}/actions/${formData.actionType}`
+    `${process.env.REACT_APP_API_ADDRESS}/system_intake/${formData.intakeId}/action`,
```

Comment:
If the ship has sailed on this I get it, but since there can be multiple actions created for a system intake, semantically the `action` in the URL should to be pluralized.